index.html

```diff
@@ -50,7 +47,263 @@ <h6>LET'S CONNECT</h6>
         
     </section>
   </section>
+<!--Wrapper to wrap around cards-->
+  <section class="wrapper">
+
+<section class="works-tonic">
+       
+       <figure class="image">
+           <img src="imageone.png" alt="imageone">
+       </figure>
+       
+       <section class="project-title">
```

Comment:
- [ ] Kindly remove the gap or blank space that is between the works and heading section.

![image](https://user-images.githubusercontent.com/55635977/164890148-2685bfd1-4dba-4962-a5fd-d222bf3d0c91.png)